### ISSUE TITLE
Authorization header rewrite causes 'jwt_auth_bad_config' on refresh and revoke

### ISSUE THREAD

**Issue by tapiopal** [jonathan-dejong/simple-jwt-authentication#68]
Hi!

We were having problems with the authorization headers not being available in php, causing errors when using any of the endpoints that require the token in the header. So, in .htaccess we added:
```
RewriteCond %{HTTP:Authorization} ^(.*)
RewriteRule .* - [e=HTTP_AUTHORIZATION:%1]
```

This makes the headers visible for the plugin and the _/token/validate_ endpoint seems to work properly, but there's something wrong with both _/token/refresh_ and _/token/revoke_. The authorization headers get sent and recognized correctly (sending the request without the headers results in 'jwt_auth_no_auth_header'), but both routes respond with an error when the token is provided in the header:
```
{
    "code": "jwt_auth_bad_config",
    "message": "JWT is not configurated properly, please contact the admin. The key is missing.",
    "data": {
        "status": 403
    }
} 
```

The key is set in wp-config.php:
`define( 'SIMPLE_JWT_AUTHENTICATION_SECRET_KEY', 'this-is-not-the-secret' );`

It seems like the secret is not set for the refresh and revoke routes, but works correctly when generating and validating tokens.

**Comment by tapiopal** [comment]
Ok. This was our mistake. We had misplaced the secret key in the wp-config.php file. What led us astray was that the token generation and validation still seemed to work perfectly.

But in the process, we did find a way to get the authorization headers without adding a RewriteRule to .htaccess. We updated the plugin code to use apache_request_headers() to get the headers, which made 'Authorization' header available in the code without editing .htaccess. This seems to work for us.
